Automattic/wp-calypso
# Reader: Filter main feed based on sidebar selection

Related to https://github.com/Automattic/loop/issues/236

## Proposed Changes

* Extends the `Recent` menu item on sidebar with the list of subscribed posts which have also made it consistent with the new designs.
* Update the `/following` API calls to include the selected `feed_id`.

## Why are these changes being made?


To enhance the filtering capabilities for feeds within the Reader's "stream" view.

## Testing Instructions



* Navigate to `/read`
* Verify that filtering is working as expected
* Navigate to `/read?flags=reader%2Frecent-feed-overhaul` and select the `stream` view
* Verify that filtering is working as expected

## Pre-merge Checklist
